src/components/Calculator.js

```diff
@@ -1,42 +1,76 @@
 import React, { Component } from "react";
+import calculate from "../logic/calculate";
+import operate from "../logic/operate";
 
 export class Calculator extends Component {
+  
+  constructor(props) {
+    super(props);
+    this.state = {calcoutput: 0,calcobj:{},calcnext:null,calctotal:null};
+  }
+  
   render() {
+    const clickbutton = (inp) => (event) => {     
+      //console.log(calculate({},this.state.calcobj))
```

Comment:
- [ ] Please remove the commented-out code.

Reply:
removed all commented code.